test/regression/cases/quality_gate_idle_all_features/experiment.yaml

```diff
@@ -46,7 +46,7 @@ checks:
     description: "Memory usage quality gate. This puts a bound on the total agent memory usage."
     bounds:
       series: total_rss_bytes
-      upper_bound: "755.0 MiB"
+      upper_bound: "775.0 MiB"
```

Comment:
I think we should make this 779 MiB

Reply:
Adjusted in [236fe38](https://github.com/DataDog/datadog-agent/pull/32272/commits/236fe3808318db673bc53f308d8528d5e5590c3b)